cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -126,7 +133,23 @@ func main() {
 	for i := 0; i < *concurrentSessions; i++ {
 		wg.Add(1)
 		go func(k int, wg *sync.WaitGroup) {
-			tc := ffmpeg.NewTranscoder()
+			var tc *ffmpeg.Transcoder
+			if *detectionFreq > 0 {
+				if detector_tc != nil {
```

Comment:
Isn't this set earlier on L115?

Reply:
We set it to `nil` a couple lines below here - so that from the 2nd session onwards we create a new session on-the-fly.

Alternatively, I think we should match it to what we do on the livepeer node - i.e. ignore the pre-initialized session at all and init a new one each time.

Will fix!